### ISSUE TITLE
RFC cli: useful commands

### ISSUE THREAD

**Issue by hachreak** [inveniosoftware/invenio-collections#36]
I'm working on cli interface and I was wondering which commands can be useful for collections.

You can see current [state of art](https://github.com/hachreak/invenio-collections/blob/cli/invenio_collections/cli.py).

I'll make a list of commands and I hope to receive many feedbacks and new ideas. Thanks.

Commands:
-  attach - Attach collection(s) to a parent.
-  create - Create new collection (you can also attach it to another collection).
-  delete - Delete a collection.
-  path  -  Print path to root.
-  query  - Print the collection query.
-  tree  -  Show the tree of the collection(s) specified.
-  update - Update an existing collection.

**Comment by hachreak** [comment]
@jirikuncar @egabancho @lnielsen @tiborsimko

**Comment by lnielsen** [comment]
I think these commands would be more useful in a flask-admin interface :-)

**Comment by jirikuncar** [comment]
@lnielsen admin interface will come when all index hooks are ready. This proposal starts with basic functionality so users can start managing their collection tree.

> - query - List of records matched by the dbquery.

IMHO this should be solved via invenio-search*. You can just print the query definition so one can combine commands:

```
inveniomanage search query "`inveniomanage collections query Demo`"
inveniomanage collections query Demo | xargs inveniomanage search query
```

**Comment by tiborsimko** [comment]
Some comments:

- `create` and `delete` seem the most useful

- `query` and `update` could just be more general `get` and `set` (for more collection attributes than just its query) -- and I agree with @jirikuncar that `query` should not run search, only list the query definition attribute

- `attach` needs `detach`

- `path` and `tree` can be perhaps merged into one CLI command with another CLI option?

**Comment by jirikuncar** [comment]
Most of the commands have been implemented in debd48b2ac20c611ac9887632392121e13338ce6.
